gruntfile.js

```diff
@@ -189,7 +189,7 @@ module.exports = function(grunt) {
         {
           expand: true,
           cwd: 'samples/web_shell/scripts/',
-          src: ['**/*.js',
+          src: ['**/*.*',
```

Comment:
`*.*` is very generic (and redundant with `*.html` on the next line). Can't we list the extensions we need explicitly?